src/components/Button/button.module.scss

```diff
@@ -471,29 +469,26 @@
     }
 
     &:hover {
-        background-color: var(--button-two-state-hover-background-color);
-        color: var(--button-two-state-hover-foreground-color);
-        outline-color: var(--button-two-state-hover-outline-color);
+        --bg: var(--green-color-10);
+        --color: var(--accent-color-70);
```

Comment:
Color in figma appears to be `#025966` which is `--bluegreen-color` or `--accent-color`